transformers/shared/constants/signature_test.go

```diff
@@ -42,6 +42,10 @@ var _ = Describe("Signature constants", func() {
 		Expect(CatFileVowSignature()).To(Equal("0xd4e8be8300000000000000000000000000000000000000000000000000000000"))
 	})
 
+	It("generates cat file box signature", func() {
```

Comment:
Should this be in a different place alphabetically? This is probably super OCD and obviously non-blocking 😅